.jenkins/Jenkinsfile-s390x

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env groovy
+
+def lib
+// Currently the loadbalancer service is not working on minikube, same for network policies
+def DEFAULT_EXCLUDED_GROUPS = "flaky,loadbalancer,networkpolicies"
+
+pipeline {
+    //agent any
+    agent{
+        node {
+            label "strimzi-s390x"
+        }
+    }
+    tools {
+        maven 'maven-3.8.4'
+    }
+    parameters {
+        string(name: 'TEST_CASE', defaultValue: "", description: 'maven parameter for executing specific tests')
+        string(name: 'TEST_PROFILE', defaultValue: 'acceptance', description: 'maven parameter for executing specific test profile')
+        string(name: 'TEST_GROUPS', defaultValue: "", description: 'maven parameter for executing specific test groups')
+        string(name: 'EXCLUDE', defaultValue: "${DEFAULT_EXCLUDED_GROUPS}", description: 'maven parameter for exclude specific test tag')
+        string(name: 'ENV_VARS', defaultValue: "", description: 'maven parameter for setting values of environment variables used in STs')
+    }
+    options {
+        timeout(time: 24, unit: 'HOURS')
+        ansiColor('xterm')
+    }
+    environment {
+        ARTIFACTS_DIR = 'systemtest/target/logs'
+        JOB_NAME_SUB = "${String.format("%.15s", JOB_NAME).toLowerCase()}"
+        ORIGIN_BASE_DIR = "${env.WORKSPACE}/origin/"
+        KUBE_CONFIG_PATH = "${env.WORKSPACE}/origin/kube-apiserver/admin.kubeconfig"
+        OPERATOR_IMAGE_PULL_POLICY = "IfNotPresent"
+        COMPONENTS_IMAGE_PULL_POLICY = "IfNotPresent"
+        JAVA_VERSION = "11"
+        TEST_HELM2_VERSION = "v2.17.0"
+        TEST_HELM3_VERSION = "v3.4.2"
+        TEST_CLUSTER = "minikube"
+        COPY_DOCKER_LOGIN = "true"
+        //Use hardcoding for ghprbCommentBody since we are not setting up GitHub PRB integration.
+        ghprbCommentBody = "ok to test"
+        JAVA_HOME = "/usr/lib/jvm/java-11-openjdk"
+        PATH = "${env.JAVA_HOME}/bin:${env.PATH}"
+        DOCKER_BUILDX = "buildx"
+        DOCKER_BUILD_ARGS = "--platform linux/s390x --load"
+        BRIDGE_IMAGE = "quay.io/strimzi/kafka-bridge:latest"
```

Comment:
What is this used for? I think the system tests should be normally using the pre-defined bridge version from the installation files and not `latest`.

Reply:
Right, makes sense.